apps/ngrx-1/src/app/app.component.ts

```diff
@@ -29,8 +30,7 @@ import { loadUsers } from './store/user/user.actions';
             *ngFor="
               let teacher of getAllTeachersForActivityType$(activity.type)
                 | async
-            "
-          >
+            ">
```

Comment:
the goal is to avoid calling a function inside a template. The view object should be constructed inside the selector. Thus in the view, only property of that object will be rendered